src/tests/GC/LargeMemory/API/gc/collect.cs

```diff
@@ -25,7 +25,7 @@ public sealed class CollectTest {
             CreateLargeObject();
         } catch (OutOfMemoryException) {
             Console.WriteLine("Large Memory Machine required");
-            return false;
+            return true;
```

Comment:
would it be possible to check that the machine indeed has small memory (which couldnt accommodate the allocation requirement? I worry that in case we have a bug that incorrectly throws OOM this might just mask that issue?

Reply:
As @jakobbotsch pointed out, the issue is not the machine, it is because multiple tests are running concurrently and they could be allocating large memory at the same time. Allocating a large object right at that point in time is the easiest way to check if we could get that much memory at that point in time.